core/src/tangram.cpp

```diff
@@ -187,11 +194,65 @@ void render() {
 
             style->onBeginDrawFrame(*m_view, *m_scene);
 
-            // Loop over all tiles in m_tileSet
             for (const auto& tile : m_tileManager->getVisibleTiles()) {
-                tile->draw(*style, *m_view);
+
+                const auto& tileID = tile->getID();
+
+                if (tile->getProxyCounter() == 0) {
+                    RenderState::stencilFunc(GL_ALWAYS, 1, 0xFF);
+                    tile->draw(*style, *m_view);
+                }
+            }
+            style->onEndDrawFrame();
+        }
+
+        // 2. Disable Color buffer writting and invert stencil value wherever depth from proxy wins
+        RenderState::colorWrite(GL_FALSE, GL_FALSE, GL_FALSE, GL_FALSE);
+        RenderState::stencilTest(GL_TRUE);
+        RenderState::stencilWrite(0xFF);
+        RenderState::stencilOp(GL_KEEP, GL_KEEP, GL_INVERT);
+        RenderState::polygonOffsetFill(GL_TRUE);
+        RenderState::polygonOffset(1.0, 1024.0);
```

Comment:
Yup, I went with 1.0 for both `slope` and `units` parameter. But it was working well for a 16bit depth buffer on my s5. 1024 was chosen as it was working best for a 16bit depth buffer (as well for a 32bit depth buffer on osx/linux). I am open to suggestions here though.